src/accel/SharedParams.hh

```diff
@@ -30,8 +30,9 @@ class StepCollector;
  * passed to a thread-local \c LocalTransporter instance. At the beginning of
  * the run, after Geant4 has initialized physics data, the \c Initialize method
  * must be called first on the "master" thread to populate the Celeritas data
- * structures (geometry, physics). \c Initialize must subsequently be invoked
- * on all worker threads.
+ * structures (geometry, physics). \c InitializeWorker must subsequently be
```

Comment:
To be specific, physics may be "physics tables" as other properties of physics (for an example, physics processes) are thread-local.

Reply:
Physics processes aren't thread-local in Celeritas: it's talking about *celeritas* thread-local data structures, not Geant4 ones.